kelraf/rafael-api
# Ch project setup 166600408

####What Does This PR Do?

1. Installation of Code Igniter Framework
2.Setup Code Igniter
3. Setup MySQL Database

#### Description of tasks to be completed?

1. Have xxamp, mamp, lamp or lamp installed on your computer.
2. Created a database with the name **rafael-api**

####How Should this be manually tested?

1. Make sure you have xxamp, mamp, lamp or lamp installed on your computer.
2. open in browser

####What are the relevant Pivotal Tracker Stories?
https://www.pivotaltracker.com/story/show/166600408
